### PR TITLE
Outreachy - Added issues suitable for small contributions

### PR DESCRIPTION
**Pull request by jpkrohling**
Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Comment by jpkrohling** [comment]
@objectiser could you please proofread ?

File: pages/outreachy.md [hunk 1]
```diff
@@ -55,6 +55,28 @@ the mailing lists or Gitter channels.
 
 Interested in becoming a mentor? Contact one of the coordinators!
 
+## Contribute
+
+As part of the application process, the Outreachy program recommends the candidates to make
```
Comment (objectiser) [inline pages/outreachy.md]: "recommends that candidates make"

Comment (objectiser) [reply pages/outreachy.md]: Only other comment is whether to use the new `good first issue` label that github is now advertising, which would be relevant after outreachy has finished?

Comment (reimertz) [reply pages/outreachy.md]: LGTM and will merge once this edit is in.  👍

Comment (jpkrohling) [reply pages/outreachy.md]: About the `good first issue`, I can certainly add that to the issues that have been tagged with `outreachy` already. Not sure if they will always be a 1 to 1, so, can we keep both for now?

Comment (jpkrohling) [reply pages/outreachy.md]: > recommends that candidates make

Done!

Comment (objectiser) [reply pages/outreachy.md]: +1

**Pull request merged**